CHANGELOG.md

```diff
@@ -4,7 +4,26 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v8.2.1](https://github.com/voxpupuli/puppet-prometheus/tree/v8.2.1) (2020-01-25)
+## [v8.2.2](https://github.com/voxpupuli/puppet-prometheus/tree/v8.2.2) (2020-02-28)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-prometheus/compare/v8.2.1...v8.2.2)
+
+**Implemented enhancements:**
+
+- Add service\_name parameter to haproxy-exporter [\#428](https://github.com/voxpupuli/puppet-prometheus/pull/428) ([unki](https://github.com/unki))
```

Comment:
This PR should update Y in x.y.z versioning 
https://semver.org/ :
> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API.

Reply:
Good catch, thank you! :)